src/main/java/Floda.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+public class Floda {
+    public static void main(String[] args) {
+        String name = "Floda";
```

Comment:
Should this be a constant? And if it was then perhaps you could have used (public static final string) and capitalised the variable name (NAME) to follow the coding standards.